olcUTIL_Geometry2D.h

```diff
@@ -1530,8 +1552,9 @@ namespace olc::utils::geom2d
 	template<typename T1, typename T2>
 	inline constexpr bool overlaps(const triangle<T1>& t1, const triangle<T2>& t2)
 	{
-		// TODO:
-		return false;
+		return overlaps(t1, t2.pos[0])
+			|| overlaps(t1, t2.pos[1])
+			|| overlaps(t1, t2.pos[2]);
 	}

```

Comment:
This is missing a case: consider t2 that contains t1

Reply:
While fixing this I've noticed contain test for circle circle works both ways (returns true as long as one contains the other). That's not the expected behavior for all right?

Reply:
Good observation about circle contains circle.

Also your overlap test for triangle-triangle still missing a case :)
https://imgur.com/a/6StWSiq

Reply:
@Piratux it's good to go now :')

Reply:
It seems like your solution will work now.
Although it can be simplified to this:

```cpp
return overlaps(t1, t2.side(0))
	|| overlaps(t1, t2.side(1))
	|| overlaps(t1, t2.side(2))
	|| overlaps(t2, t1.pos[0]);
```